freqtrade/optimize/backtesting.py

```diff
@@ -280,8 +280,8 @@ def load_bt_data_detail(self) -> None:
 
             if unavailable_pairs:
                 raise OperationalException(
-                        f"Pairs {', '.join(unavailable_pairs)} got no leverage tiers available. "
```

Comment:
@xmatthias Should I revert style changes too?

Reply:
it doesn't belong here - HOWEVER - it's just one line, so it's fine here.

It is however very problematic if a whole file changes (most every line) due to formatting changes - as it makes identifying "real" changes near impossible / very timeconsuming - drawing away the focus from reviewing the changed logic (and how it ties into all the rest of the bot) to "is here a change in logic - or just in formatting".